src/server/components/serializers/FlettefeltSerializer.tsx

```diff
@@ -42,6 +48,12 @@ export const FlettefeltSerializer = (props: IFlettefeltSerializerProps) => {
         ))}
       </ul>
     );
+  } else if (erFlettefeltFritekstfelt) {
+    return (
+      <span className={høyrestill ? 'høyrestill' : ''}>
+        {flettefelt[0]?.split('\n\n').map((avsnitt, i) => <StyledDiv key={i}>{avsnitt}</StyledDiv>)}
```

Comment:
Trenger vi å kjøre split? 

Vil ikke `<StyledDiv>{flettefelt[0]}</StyledDiv>` ivareta linjeskiftene så lenge `StyledDiv` har `pre-wrap` ? 

Eller er det noe jeg ikke forstår, kanskje?

Reply:
Med `StyledDiv` og sånn det er gjort i `FritekstområdeSerializer`:
```html
<span style={{ whiteSpace: 'pre-wrap' }}>{avsnitt.innhold}</span>
```
blir linjeskiftene fjernet. Funker linjeskift for dere der dere bruker `FritekstområdeSerializer`?

Hvis du har forslag til en bedre måte å gjøre det på, unngår jeg veldig gjerne split

Reply:
Det høres rart ut. Ser ut som `FritekstområdeSerializer ` funker bra når jeg prøver i vår løsning. 

Jeg har iallefall brukt `pre-wrap` rett i en `span` for å ivareta linjeskift tidligere. 

<img width="1178" alt="Screenshot 2025-01-10 at 12 53 00" src="https://github.com/user-attachments/assets/fd8b656d-8c0a-4801-83ca-508456441678" />